spec/setup_acceptance_node.pp

```diff
@@ -10,6 +10,32 @@
     $ssl_cert_source = '/etc/pki/tls/certs/localhost.crt'
   }
 }
+if $facts['os']['family'] == 'RedHat' and versioncmp($facts['os']['release']['major'], '8') >= 0 {
+  exec { 'gencerts':
+    command     => '/usr/libexec/httpd-ssl-gencerts',
+    refreshonly => true,
+    subscribe   => Package[$package_name],
+    before      => [
+      File["/etc/ldap/ssl/${::fqdn}.key"],
```

Comment:
please dont use the legacy topscope facts. please use the facts hash and networking.fqdn

Reply:
This has been fixed.